index.js

```diff
@@ -45,6 +45,7 @@ process.stdin.on("data", function (data) {
     }
 });
 
+var net = require('net');
```

Comment:
It's not unused. It's used to to validate user input, which was your suggestion over a regex.